internal/pkg/plugininstaller/reposcaffolding/option.go

```diff
@@ -10,12 +10,13 @@ import (
 const (
 	transitBranch      = "init-with-devstream"
 	appNamePlaceHolder = "_app_name_"
+	defaultCommitMsg   = "init with devstream"
 )
 
 type Options struct {
-	RepoType        string  `validate:"oneof=gitlab github" mapstructure:"repo_type"`
-	SourceRepo      SrcRepo `validate:"required" mapstructure:"source_repo"`
-	DestinationRepo DstRepo `validate:"required" mapstructure:"destination_repo"`
+	RepoType        string   `validate:"oneof=gitlab github" mapstructure:"repo_type"`
```

Comment:
The doc says that `SourceRepo` only supports `GitHub` now, so in the future it may also support `GitLab`. The `RepoType` can be placed inside `SrcRepo` and `DstRepo` to meet future needs.